tools/deployment/package-helm/templates/results-cache-data-pv.yaml

```diff
@@ -1,9 +1,9 @@
 {{- include "clp.createLocalPv" (dict
   "root" .
-  "name" (printf "%s-results-cache-data" (include "clp.fullname" .))
-  "component" "results-cache"
+  "component_category" "results-cache"
+  "name" "data"
   "nodeRole" "control-plane"
   "capacity" "20Gi"
   "accessModes" (list "ReadWriteOnce")
-  "hostPath" (printf "%s/results-cache" .Values.clpConfig.data_directory)
+  "hostPath" (printf "%s/results_cache" .Values.clpConfig.data_directory)
```

Comment:
sorry, the `mkdir` directory should also be updated but was missed:
[results_cache](https://github.com/y-scope/clp/blob/1307032802d10b7003261775bd6f77229913a3fe/tools/deployment/package-helm/test.sh#L11)
-> `results_cache`

@coderabbitai create an issue to track